packages/ui-side-nav-bar/CHANGELOG.md

```diff
@@ -0,0 +1,891 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [8.38.1](https://github.com/instructure/instructure-ui/compare/v8.38.0...v8.38.1) (2023-06-13)
+
+**Note:** Version bump only for package @instructure/ui-side-nav-bar
```

Comment:
I'd remove all the text from here since SideNavBar will start to exist in the next version